docs/standard/serialization/system-text-json/snippets/how-to/csharp/JsonDocumentDataAccess.cs

```diff
@@ -70,5 +74,15 @@ private static void AverageGrades_Alternative(string jsonString)
             Console.WriteLine($"Average grade : {average}");
             // </AverageGrades2>
         }
+
+        private static void Compare()
+        {
+            // <DeepEquals>
+            JsonElement left = JsonDocument.Parse("10e-3").RootElement;
+            JsonElement right = JsonDocument.Parse("0.01").RootElement;
```

Comment:
This is a mistake I introduced, should be:
```suggestion
            JsonElement right = JsonDocument.Parse("0.001").RootElement;
```